<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="ChatBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>771</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="stat">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Глянуть статистику</string>
     </property>
    </widget>
    <widget class="QPushButton" name="check">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Проверить ответ</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textArea">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>431</width>
       <height>401</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Подключение</string>
    </property>
    <addaction name="separator"/>
    <addaction name="ConnectToChat"/>
    <addaction name="LeaveChat"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ConnectToChat">
   <property name="text">
    <string>Подключиться к чату</string>
   </property>
  </action>
  <action name="LeaveChat">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Выйти из чата</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
